### DISCUSSION TITLE
"Anything" with "Full Examples"

### DISCUSSION THREAD

**Discussion by ijpatricio** [serversideup/docker-php#343] [Ideas & Feature Requests]
Hey @jaydrogers , and everyone interested!

I believe the fundamentals are in place! Also, there's something missing.

A full "grab n go" example. Something like templates in the self-hosting world, where you clone a repo, go to some folder, and it's ready to use just by "docker compose up".

### What?

Let's suppose:

  - I want to have an app with "webapp, cron, queue, pgsql, websockets"
  - I might be a Docker seasoned person, but I'm also lazy, and like to have ready things
  - I might be a Docker beginner, or just thinking about adopting

I would like to grab something and go.

- In the S6 fashion.
- Or in a "one service per executable" fashion.

Hey, I know the number combinations of app components, and docker-php flavour combinations, is wild!!
But we got to do something, at least a basic thing, no?!

### Why?

There's not like even a couple of starting points! I know `spin` exists, but IMO, it will solve a different problem.

### How?

I'm not sure if it would be better to have it like:
- fully functioning apps
  - CON: gets old with time
  - PRO: act as skeleton, and can be tested in CI
- just compose files, Dockerfiles and extras, like a .env or a script file
  - CON: it's for a smaller niche. might get outdated by surprise  
  - PRO: simpler to have, easier to maintain bc act as a snippet bucket

### Conclusion

This is nothing like set on stone, everything is mutable, just "braindumping", so everyone can chime in.

**Comment by jaydrogers** [comment]
We have something in the works. Stay tuned 🤓

I’ll work on getting some simple examples up too 👍

**Reply by sawirricardo** [reply]
Any update about this? 👍

**Reply by jaydrogers** [reply]
@sawirricardo Have you seen this? https://www.youtube.com/watch?v=5z2JoEt5XIk

**Reply by sawirricardo** [reply]
> @sawirricardo Have you seen this? https://www.youtube.com/watch?v=5z2JoEt5XIk

Hey! Got a time to look into this and i think it's awesome! 

Do you have a guide to use spin with synology nas?
